### ISSUE TITLE
[Request] Double-space period

### ISSUE THREAD

**Issue by samantohermes** [rkkr/simple-keyboard#125]
An option for a double tap on space bar to insert a period followed by a space.

**Comment by abh153k** [comment]
+1. I miss this one from ios

**Comment by JM480** [comment]
I think this feature would fit this keyboard pretty well.

**Comment by Chimkentus** [comment]
Yes, it would be convenient with an optional automatic space after punctuation feature. Anyone who has to carefully type a lot of text knows what I mean. I think all users will appreciate this.

**Comment by Chimkentus** [comment]
I followed the ThHareau link discussing this problem and realized that you are complicating it a little: you don’t need any “Quick period + space”. In the settings of your keyboard there is a setting “Capitalize automatically - write the first word in a sentence with a capital letter” - it should be similar for this function, make a separate function “Space after punctuation”, which can be enabled if desired. There are only a few punctuation marks after which you are supposed to leave a space (this is . , : ; ! ? I may have missed something, but if necessary, I’ll add it), but we use them quite often.
